README.md

```diff
@@ -1,65 +1,73 @@
-README
-======
+<p align="center"><a href="https://symfony.com" target="_blank">
+    <img src="http://symfony.com/logos/symfony_black_02.svg">
```

Comment:
What about adding logo to resources, for example in `FrameworkBundle` and don't rely on external content? There is Symfony svg in `WebProfilerBundle` but not black and smaller.